README.md

```diff
@@ -1 +1,62 @@
-# mag-calculator
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name: Bookstore
+
+> Description the project: This project is for Magiciant Calculator. ;)
+
+
+## Built With
+
+- Major languages
+> Html
+> CSS
+> JavaScript
+> React
+
+
+## Authors
+
+👤 **Sajjad Rajaee**
+
+- GitHub: [@sajjadrajaee](https://github.com/sajjadrajaee)
+- Twitter: [@sajjadrajaee](https://twitter.com/sajjadrajaee)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/sajjadrajaee)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+- clone the project from GitHub: [@sajjadrajaee](https://github.com/sajjadrajaee/mag-calculator)
+- run the code on a text editor
+
+### Prerequisites
+
+no neaded 
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you also update the `### Prerequisites` sub-section content by adding there some tools required to clone and run your project locally(e.g.: **Git**, **Node**, **IDE**,...)